src/common/options/osd.yaml.in

```diff
@@ -133,6 +133,54 @@ options:
   - osd_recovery_sleep
   flags:
   - runtime
+- name: osd_recovery_sleep_degraded
+  type: float
+  level: advanced
+  desc: Time in seconds to sleep before next recovery or backfill op when pg is degraded.
+    This setting overrides _ssd, _hdd, and _hybrid if non-zero.
+  fmt_desc: Time in seconds to sleep before the next recovery or backfill op when pg
+    is degraded. Increasing this value will slow down recovery operation while client
+    operations will be less impacted.
+  default: 0
+  flags:
+  - runtime
+  with_legacy: true
+- name: osd_recovery_sleep_degraded_hdd
+  type: float
+  level: advanced
+  desc: Time in seconds to sleep before next recovery or backfill op for HDDs
+    when pg is degraded.
+  fmt_desc: Time in seconds to sleep before next recovery or backfill op
+    for HDDs when pg is degraded.
+  default: 0.1
```

Comment:
IMO this should be smaller than its osd_recover_sleep_hdd counterpart. May be 0.05?